### DISCUSSION TITLE
Urgenza

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#105] [Richieste di aiuto] [answered]
Gentile Dessì,la prego di contattarmi  privatamente alla seguente mail per un problema  :

**Comment by trinko** [comment, marked as answer]
Mi spiace, ma questo non è un servizio di assistenza.
Se è un problema generale, prova a esporlo e qualcuno, se ne avrà il tempo, cercherà di aiutarti, nei limiti del possibile.
